test/unit_tests/utils/calculation_test.js

```diff
@@ -0,0 +1,26 @@
+/* global describe it */
+
+const chai = require( 'chai' );
+const expect = chai.expect;
+const calculation = require( '../../../src/js/utils/calculation' );
+
+describe( 'calculation', () => {
+
+  describe( 'getFirstNumber', () => {
+
+    it( 'should return the first number in an array of values', () => {
+      const valToCheck = 2;
+      const mockArray = [
+        [ 'one', 'one' ],
+        [ 'two', valToCheck ],
+        [ 'three', '3' ]
+      ];
+      expect( calculation.getFirstNumber( mockArray ) ).to.equal( valToCheck );
```

Comment:
if I pass this array `[ 'two', valToCheck ]` instead of `mockArray` then the test fails with:

`     AssertionError: expected false to equal 2`

does the function only expect nested arrays?

Reply:
Yes, that's what this TODO was trying to say https://github.com/cfpb/cfpb-chart-builder/pull/121/files#diff-f543c19dcfe072e9cf7ddce186eff78cR5